src/core/usecases/runningService.ts

```diff
@@ -135,6 +140,38 @@ export const thunks = {
 
             dispatch(actions.isUserWatchingChanged({ isUserWatching }));
         },
+    "upgradeService":
+        (serviceId: string): ThunkAction<void> =>
+        async (...args) => {
+            const [dispatch, getState] = args;
+
+            {
+                const state = getState().runningService;
+
+                if (state.isUpdating) {
+                    return;
+                }
+            }
+            const restorableRunningPackageConfigs =
+                runningPackageConfigSelectors.restorableRunnigPackageConfigs(getState());
+            const restorableRunningPackageConfig = restorableRunningPackageConfigs.find(
+                e => e.runningPackageConfig.name === serviceId
+            )?.runningPackageConfig;
+            assert(restorableRunningPackageConfig !== undefined);
```

Comment:
You didn't take the time to propagate your renaming.  
I have the context so I can figure out what's going on but it will be criptic for the next guy.